SPECS/glib/glib.spec

```diff
@@ -1,52 +1,54 @@
-Summary:	Low-level libraries useful for providing data structure handling for C.
-Name:		glib
-Version:	2.58.0
-Release:        6%{?dist}
-License:	LGPLv2+
-URL:		https://developer.gnome.org/glib/
-Group:		Applications/System
+%define sha1 glib=c00e433c56e0ba3541abc5222aeca4136de10fb8
```

Comment:
Please remove the `sha1` macro while you're here. It's a leftover we no longer need and remove whenever we get a chance.